tools/topology/sof/pipe-volume-demux-playback.m4

```diff
@@ -86,7 +148,9 @@ W_BUFFER(2, COMP_BUFFER_SIZE(DAI_PERIODS,
 
 P_GRAPH(pipe-ll-playback-PIPELINE_ID, PIPELINE_ID,
 	LIST(`		',
-	`dapm(N_BUFFER(0), N_PCMP(PCM_ID))',
+	`dapm(N_BUFFER(3), N_PCMP(PCM_ID))',
+	`dapm(N_CODEC_ADAPTER(0), N_BUFFER(3))',
+	`dapm(N_BUFFER(0), N_CODEC_ADAPTER(0))',
```

Comment:
the pipe-volume-demux-playback.m4 is quite generic, adding codec_adapter to this makes confusion, as @cujomalainey commented, create a new .m4 for it is better choice.